.cargo/config.toml

```diff
@@ -0,0 +1,16 @@
+[env]
+CC_aarch64_unknown_linux_musl = "aarch64-linux-musl-gcc"
+CC_x86_64_unknown_linux_musl = "x86_64-linux-musl-gcc"
+
+# Zydis CMake config requires a C++ compiler for the test cases but the musl
+# toolchain doesn't have one. We simply force the compiler to `g++` here to
+# trick CMake into passing the check for a C++ compiler: it isn't actually
+# used anyway.
+TARGET_CXX = "g++"
+
+[target.aarch64-unknown-linux-musl]
+# The vendored OpenSSL fails to link with GNU ld on aarch64+musl.
```

Comment:
Is this needed or is it a leftover from elastic internal workspace? Looks like symblib does not link to openssl at the moment.

Reply:
Good catch, yeah: it's a leftover.

Reply:
removed with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/360/commits/c6a04910ebcab64f5ffbcc145e1a38aca8a3e9bf